common/djangoapps/student/migrations/0013_historicalcourseenrollment_history_change_reason.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('student', '0012_sociallink'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='historicalcourseenrollment',
+            name='history_change_reason',
+            field=models.CharField(max_length=100, null=True),
+        ),
+    ]
```

Comment:
@edx/devops While the rest of these migrations should cause no troubles, this particular migration is problematic, adding a field to a table with 47 million rows (according to the read replica):
```
 mysql> select min(id) from student_historicalcourseenrollment;
+---------+
| min(id) |
+---------+
|      56 |
+---------+
1 row in set (0.00 sec)

mysql> select max(id) from student_historicalcourseenrollment;
+----------+
| max(id)  |
+----------+
| 47823634 |
+----------+
1 row in set (0.00 sec)
```
How should this migration be handled? The migration was triggered by the version bump in django-simple-history, which was necessary to add Django 1.11 support.

Here's the row counts for the rest of the tables:
```
mysql> select count(*) from verify_student_historicalverificationdeadline;
+----------+
| count(*) |
+----------+
|    14774 |
+----------+
1 row in set (0.02 sec)

mysql> select count(*) from api_admin_historicalapiaccessrequest;
+----------+
| count(*) |
+----------+
|      624 |
+----------+
1 row in set (0.00 sec)

mysql> select count(*) from microsite_configuration_historicalmicrositetemplate;
+----------+
| count(*) |
+----------+
|        0 |
+----------+
1 row in set (0.00 sec)

mysql> select count(*) from microsite_configuration_historicalmicrositeorganizationmapping;
+----------+
| count(*) |
+----------+
|        0 |
+----------+
1 row in set (0.00 sec)

mysql> select count(*) from credit_historicalcreditrequest;
+----------+
| count(*) |
+----------+
|     3551 |
+----------+
1 row in set (0.01 sec)

mysql> select count(*) from credit_historicalcreditrequirementstatus;
+----------+
| count(*) |
+----------+
|  1552070 |
+----------+
1 row in set (2.44 sec)
```

Reply:
As discussed with DevOps, I've removed this commit containing the django-simple-history upgrade for now.